auth_saml/static/description/index.html

```diff
@@ -156,6 +156,13 @@ <h3>Contributors</h3>
               >bodedra&#64;opensourceintegrators.com</a
             >&gt;
           </li>
+          <li>
+            Torvald Baade Bringsvor &lt;<a
+              class="reference external"
+              href="mailto:torvald&#64;flytconsulting.no"
+              >torvald&#64;flytconsulting.no</a
+            >&gt;
+          </li>
```

Comment:
This file is compiled from the elements in the `readme/` directory by a bot. Change `readme/CONTRIBUTORS.md` directly too, otherwise your changes will be removed.

Reply:
Aha! Ok, added in CONTRIBUTORS.